com.unity.render-pipelines.universal/Shaders/Lit.shader

```diff
@@ -85,6 +91,7 @@ Shader "Universal Render Pipeline/Lit"
             HLSLPROGRAM
             #pragma exclude_renderers d3d11_9x gles
             #pragma target 4.5
+            //#pragma enable_d3d11_debug_symbols
```

Comment:
should be removed.

Reply:
Fixed. Personally I would prefer having //#pragma enable_d3d11_debug_symbols in every URP shader. As in case of debuging it is much simplier just to uncomment it, instead of writing again and again.